docs/README.md

```diff
@@ -64,3 +64,92 @@ The identifier should match the name, with these additional guidelines:
   - 👎 Not recommended: `fullscreen-api`
   - 👍 Recommended: `user-pseudos`
   - 👎 Not recommended: `user-valid-and-user-invalid`
+
+### Descriptions
+
+* Describe, in the active voice, what a feature does or is.
+  Think about how developers will use it, not abstract technology relationships.
+  Start with a template like this:
+
+  - `The <property> <sets> the <noun>.`
+  - `The <interface> <verbs> the <noun>.`
+  - `The <type> represents <nouns>.`
+  - `<format> is a <kind> or <variety>`.
+
+* Description text must stand alone.
+  It should not refer to text, images, or other content outside the short description.
+  Try reading the sentence aloud.
+  Does it still make sense without mentioning the name or ID?
+
+* Start descriptions with words that are distinct to the feature.
+  For example, prefer "The `some-prop` CSS property…" and avoid "The CSS property `some-prop`…."
+
+* Avoid circular descriptions.
+  For example, prefer "The `filter()` method returns the items…" over "The `filter()` method filters the items…."
+
+* Never mention support or standards status.
+  This information *will* go out of date and sooner than you think.
+
+* For every rule, there's a counterexample.
+  Use your best judgement before writing something absurd.
+
+* See the [word and phrase list](#word-and-phrase-list) for specific guidelines.
+
+#### Word and phrase list
+
+##### allows
+
+OK in usage such as "allows you to…."
+Avoid where there is no named actor, as in "the feature allows magic to happen."
```

Comment:
I think three things are going on here:

1. "allows" isn't great, but is difficult to avoid completely. Some things just aren't obvious and we have to invoke developers applications, rather than describing something in strictly functional terms. There are alternatives (provides, enables, etc.) but we should use this one specifically so it's consistent and easier to develop shared expectations around.

1. If you're going to use "allows" then you need to do it in a specific way to avoid passive or indirect descriptions. That is, you can't say "allows" without answering the question, "who allows it?" Compare "The `filter()` method allows arrays to be copied…" against "The `filter()` method allows you to copy arrays…". It's much harder to end up with weak constructions like "to be copied", if we forbid actor-less descriptions.

1. It's OK to refer to developers directly (i.e., "`filter()` allows **you** to copy…"), when it makes sense to do so. I've created a specific entry for that in the more general guidelines at the top, with 1a09c0f0472fb078aabd63eb3fcdb68baa762b41.